docs/data/material/components/pickers/deprecated-pickers.md

```diff
@@ -0,0 +1,19 @@
+# Date / Time pickers
+
+<p class="description">Now part of MUI X, a collection of advanced components.</p>
+
+## What is the reason to move the Pickers to MUI X?
+
+MUI X components are those meant to support complex use cases. Pickers can seem trivial but they're considerably tricky to pull off correctly, so we've moved them over to the X collection to make sure we're investing proper time and effort. If you want to know more details, check out the [blog post](/blog/lab-pickers-to-mui-x/).
```

Comment:
Cut unnecessary wording, rephrase to be less "we"-centric, make the link text more descriptive.

```suggestion
MUI X components are built for complex use cases. Date and time pickers are difficult to pull off correctly, so they deserve the special attention that they will receive as part of MUI X. To learn more, check out the [blog post about the move](/blog/lab-pickers-to-mui-x/).
```